### PR TITLE
isl29125: added interrupt

### PR DESCRIPTION
**Pull request by cgundogan**
Adopted from #6118 because of inactivity.

### CODE REVIEW

**Comment by cgundogan** [comment]
@smlng wanna test? (:

File: tests/driver_isl29125/main.c [hunk 1]
```diff
@@ -53,6 +66,17 @@ int main(void)
         return 1;
     }
 
+    puts("Initializing interrupt");
+    if (isl29125_init_int(&dev, ISL29125_INTERRUPT_STATUS_RED,
```
Comment (smlng) [inline tests/driver_isl29125/main.c]: rewrite and save the `else`:
```
if (isl29125_init_int() != 0) {
    puts("[FAILED]");
    return 1;
}
```

Comment (cgundogan) [inline tests/driver_isl29125/main.c]: >@smlng wanna test? (:

wow that was quick.

**Comment by cgundogan** [comment]
@smlng addressed your comment and also took the liberty to squash

**Comment by smlng** [comment]
tested with samr21-xpro: works. ACK and GO, when Murdock happy, too!

**Comment by cgundogan** [comment]
oh c'mon murdock :/ it seems the check for the needs squashing label is done at the beginning of the build

**Pull request merged**